src/borg/archiver/rinfo_cmd.py

```diff
@@ -48,9 +46,31 @@ def do_rinfo(self, args, repository, manifest, cache):
                     location=repository._location.canonical_path(),
                     version=repository.version,
                     append_only=repository.append_only,
-                    **info,
+                    encryption=info["encryption"],
                 )
             )
+
+            response = repository.info()
+            storage_quota = response["storage_quota"] or None
```

Comment:
So you intended to use None instead of 0, but you do not really make a difference in the code below.

Reply:
Also, it would be interesting how much space a repo uses, even if there is not a set limit via storage_quota.